.freeCodeCamp/tooling/hot-reload.js

```diff
@@ -9,22 +9,21 @@ const { testPollingRate, runTestsOnWatch } = await getProjectConfig(
   currentProject
 );
 
-function hotReload(ws) {
+const defaultPathsToIgnore = [
```

Comment:
Looks like this list is in here twice - just pointing it out. They may both be needed.

Reply:
This is only in here once...

Reply:
1 here, 2 here:
https://github.com/freeCodeCamp/freeCodeCampOS/pull/118/files#diff-da1b8b73b6e95fe493931772fd5c7b532c68dfc1454c88c0d0bc33dd7c06d2d6R52

same list of files